test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/RiveAnimationFollowTouch.cs

```diff
@@ -8,7 +9,7 @@ public class RiveAnimationFollowTouch : IExample
         private Window window;
         private Layer defaultLayer;
 
-        RiveAnimationView rav;
+        Tizen.NUI.Extension.RiveAnimationView rav;
```

Comment:
you can just use like this. because you already add 'using Tizen.NUI.Extension'
```csharp
RiveAnimationView rav;
```

Reply:
currently, we are moving RiveAnimationView from Tizen.NUI to Tizen.NUI.Extension.
until finishing this work, I think Tizen.NUI.Extension prefix is necessary.